utils/build-script-impl

```diff
@@ -2039,6 +2051,15 @@ for host in "${ALL_HOSTS[@]}"; do
                     )
                 fi
 
+                if contains_product "lldb" ; then
```

Comment:
I think just `if [[ ! "${SKIP_BUILD_LLDB}" ]] ; then` is OK.
(no need to create new `contains_product()` function)